sds_data_manager/lambda_code/SDSCode/dependency_config_codice.py

```diff
@@ -0,0 +1,383 @@
+"""Stores the downstream dependency configuration of CoDICE.
+
+This is used to populate pre-processing dependency table in the database.
+
+NOTE: This setup assumes that we get one data file everyday with all
+the data of multiple apid. This is why we have only one file for each
+l0 and descriptor is raw. And l1a that depends on l0 has 'all' as
+descriptor with assumption that l1a could produce multiple files
+with different descriptor. Those different descriptor are handle in its own
+code in imap_processing repo.
+"""
+
+from .database.models import PreProcessingDependency
+
+downstream_dependents = [
+    # l0 to l1a
+    PreProcessingDependency(
+        primary_instrument="codice",
+        primary_data_level="l0",
+        primary_descriptor="raw",
+        dependent_instrument="codice",
+        dependent_data_level="l1a",
+        dependent_descriptor="all",
+        relationship="HARD",
+        direction="DOWNSTREAM",
+    ),
+    # l1a to l1b
+    PreProcessingDependency(
+        primary_instrument="codice",
+        primary_data_level="l1a",
+        primary_descriptor="hskp",
```

Comment:
It's ok right now. Something we can bring up during nomenclature discussion in the future. Should we use `hk` or `hskp`?

Reply:
Yeah, I agree, something we should discuss. I know that CoDICE uses `hskp` all throughout their algorithm document ...